Celeste.Mod.mm/MonoModRules.cs

```diff
@@ -382,11 +382,18 @@ static class MonoModRules {
                 version = new Version(versionInts[0], versionInts[1], versionInts[2], versionInts[3]);
             }
 
+            // Check if Celeste version is supported
             Version versionMin = new Version(1, 3, 1, 2);
             if (version.Major == 0)
                 version = versionMin;
             if (version < versionMin)
-                throw new Exception($"Unsupported version of Celeste: {version}");
+                throw new Exception($"Unsupported version of Celeste: {version}. The minimum required version of Celeste is: {versionMin}.");
```

Comment:
The minimum required version was not shown specifically to throw off any pirates unable to find the actually latest version.